pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -102,39 +102,39 @@ type Results struct {
 	PodErrors     map[*v1.Pod]error
 }
 
-// AllNonPendingPodsScheduled returns true if all of the non-pending pods scheduled.  This is useful in consolidation as
-// we don't care if a pod was pending before consolidation and will still be pending after. It may be a pod that we
-// can't schedule at all and don't want it to block consolidation.
+// AllNonPendingPodsScheduled returns true if all pods scheduled.
+// We don't care if a pod was pending before consolidation and will still be pending after. It may be a pod that we can't
+// schedule at all and don't want it to block consolidation.
 func (r Results) AllNonPendingPodsScheduled() bool {
```

Comment:
This looks good to me. Just a thought: I wonder if we could just have a filter on the results so that we could do something like

```go
results.NonPendingPods().Errors()
results.PendingPods().Errors()
results.Errors()
```

Reply:
Since we only care about the NonPendingPods in code, I don't think there's a use in adding these, but could definitely see the use in the future.